test/e2e/domainmapping/domain_mapping_websocket_test.go

```diff
@@ -0,0 +1,127 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package domainmapping
+
+import (
+	"context"
+	"net/url"
+	"strings"
+	"testing"
+
+	"knative.dev/pkg/test/spoof"
+
```

Comment:
Is there any reason for the break line for L#29 and L#36? If not, let's clean them up.